roles/openshift_docker_registry/tasks/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+- set_fact: _option_images="--images={{ openshift_registry_url|quote }}"
+  when: openshift_registry_url is defined
+
+- name: Deploy OpenShift Docker Registry
+  command: openshift admin registry --create --credentials="{{ansible_env['HOME']}}"/.config/openshift/config {{ _option_images|default() }}
```

Comment:
This actually bit me today...  and @deads2k schooled me on this.  We should really be creating these with the openshift-registry creds (and the router with the openshift-router creds) in /var/lib/openshift/openshift.local.certificates/

my apologies for leading you astray on using the copied openshift-client configs.